misc/ls-extensions/modules/bal-shell-service/src/test/resources/testcases/errors.json

```diff
@@ -0,0 +1,38 @@
+[
+  {
+    "source": "int x = \"10\";",
+    "result": {
+      "errors":["Invocation of the snippet failed."],
+      "diagnostics":["error: incompatible types: expected \u0027int\u0027, found \u0027string\u0027\n\tint x \u003d \"10\";\n\t        ^--^","Compilation aborted due to errors."],
+      "metaInfo":{"definedVars":[],"moduleDclns":[]},
+      "consoleOut":""
+    }
+  },
+  {
+    "source": "int string x = 10;",
+    "result": {
+      "errors":["Parsing the node from the statement failed."],
+      "diagnostics":["syntax error: invalid token \u0027string\u0027\n\tint string x \u003d 10;\n\t    ^----^","Parsing aborted due to errors."],
+      "metaInfo":{"definedVars":[],"moduleDclns":[]},
+      "consoleOut":""
+    }
+  },
+  {
+    "source": "1/0",
+    "result": {
+      "errors":["panic: error(\"{ballerina}DivisionByZero\",message\u003d\" / by zero\")"],
+      "diagnostics":["Execution aborted due to unhandled runtime error."],
+      "metaInfo":{"definedVars":[],"moduleDclns":[]},
+      "consoleOut":""
+    }
+  },
+  {
+    "source": "service hello on new http:Listener(9090) { }",
+    "result": {
+      "errors":["Converting the node into a snippet failed."],
+      "diagnostics":["Services are not allowed within REPL."],
```

Comment:
It seems that we are using shell warning/error messages. What will be the notebook output if we try to use `main()` method ?

Reply:
```text
Parsing the node from the statement failed.
Function name 'main' not allowed in Ballerina Shell.
```
At the moment we are returning errors generate by the shell as it is, will be translated for the notebook at the VSCode plugin implementation.

Reply:
I think here we have two options.

IMO notebook is also REPL based system and we can change shell error/warning messages mentioning `not allowed in REPL` so error messages will valid in both shell and the notebook.

Other option is to define separate notebook specific errors in the notebook implementation

@mohanvive WDYT ?

Reply:
Added a new test set to address the use of `main()` in [42c86c2](https://github.com/ballerina-platform/ballerina-lang/pull/35871/commits/42c86c21554f68fb284b9c21969783fe3bf13ba8).